src/recorder.js

```diff
@@ -155,16 +155,19 @@ Recorder.prototype.pause = function( flush ) {
   if ( this.state === "recording" ) {
     this.state = "paused";
     if ( flush && this.config.streamPages ) {
+      var encoder = this.encoder;
       return new Promise((resolve, reject) => {
-        const callback = (e) => {
+        var callback = (e) => {
           if ( e["data"]["message"] === 'flushed' ) {
-            this.encoder.removeEventListener( "message", callback );
+            encoder.removeEventListener( "message", callback );
+            encoder = null;
+            callback = null;
```

Comment:
Is this necessary?  The shouldn't be any issues garbage collecting these objects if they are unreferenced.

Reply:
@tomwidmer Yes, please remove, unless you have some evidence we are leaking memory if we don't do this.

Reply:
Done (in both places)